apps/transmitmedia.cpp

```diff
@@ -911,7 +914,6 @@ class UdpCommon
             // is called with multicast address. Write the address
             // that designates the network device here.
             // Also, sets port sharing when working with multicast
-            sadr = maddr;
             int reuse = 1;
             int shareAddrRes = setsockopt(m_sock, SOL_SOCKET, SO_REUSEADDR, reinterpret_cast<const char*>(&reuse), sizeof(reuse));
```

Comment:
Check if this is still necessary to be set before setting the multicast option, or can be moved to the place directly before binding so that the Windows-specific code is in one place.

Reply:
Ok, confirmed here: setting SO_REUSEADDR is not necessary because the same option is set the same way in the same function few lines earlier, this time for any platform, so this part setting the option can be removed. So please, as you are changing things here, remove this all, and then place the Verb() calls in appropriate conditionals before the `::bind()` call.